### DISCUSSION TITLE
Server refuses to allocate more memory

### DISCUSSION THREAD

**Discussion by pact0** [Infinidoge/nix-minecraft#114] [Q&A] [answered]
Hey, my fabric server refuses to allocate more ram. I used spark profiler to check the performance and I noticed it only had 2 gigs allocated instead of 8 as specified. Same thing is visible for example in htop. This is my config.
```
{
       enable = true;
       autoStart = true;
       enableReload = false;
       package = pkgs.fabricServers.fabric-1_21;
       serverProperties = {
         gamemode = 0;
         difficulty = 1;
         simulation-distance = 4;
         view-distance = 7;
         #level-seed = 8;
         restart = "always";
         server-port = 25565;
         jvmOpts = "-Xms8192M -Xmx8192M";
       }
}
```

**Comment by Infinidoge** [comment, marked as answer]
`jvmOpts` doesn't go in `serverProperties`. It goes in the same level as `package`, etc.

**Comment by pact0** [comment]
You're right, thanks, I somehow missed that it was in the wrong scope
